bindings/python/pyiree/rt/vm.cc

```diff
@@ -197,8 +197,11 @@ std::string VmVariantList::DebugString() const {
   absl::StrAppend(&s, "<VmVariantList(", size(), "): [");
 
   for (iree_host_size_t i = 0, e = size(); i < e; ++i) {
-    iree_vm_variant_t* variant =
-        iree_vm_variant_list_get(mutable_this->raw_ptr(), i);
+    iree_vm_variant2_t variant = {0};
+    if (iree_status_is_ok(
+            iree_vm_list_get_variant(mutable_this->raw_ptr(), i, &variant))) {
+      return "Error";
```

Comment:
Could use a more descriptive error string here

How about this?

```suggestion
      return "[Error getting variant from list]";
```

Or should this
```
absl::StrAppend(&s, "Error");
continue;
````
rather than break out of the loop? Is it possible to continue, or does an error indicate that future elements will also fail?